components/identities/actions/identityActions.tsx

```diff
@@ -149,7 +153,10 @@ const IdentityActions: FunctionComponent<IdentityActionsProps> = ({
   useEffect(() => {
     if (isAutoRenewalEnabled) {
       setDisableRenewalCalldata(
-        autoRenewalCalls.disableRenewal(callDataEncodedDomain[1].toString())
+        autoRenewalCalls.disableRenewal(
+          autoRenewalContract,
+          callDataEncodedDomain[1].toString()
```

Comment:
Some questions raised after reading this code:
- Can I enable two auto-renewals?
- What happens if I do? 
- Could we debit two times? 
- If I have two enabled could I enable the other one through the front-end? 
- If I have 2 enabled could I disable 2 in one TX (multicall)?

Reply:
the frontend should not let you enable 2 auto renewals, if you do we only take the most recent auto renewal when we fetch it and process it in the auto renewal bot. 
We cannot debit 2 times for the same domain in one year there is a safety check in the contract. 

`If I have two enabled could I enable the other one through the front-end?`> I didn't understand this case

`If I have 2 enabled could I disable 2 in one TX (multicall)` I didn't handle this case as it's not supposed to happen, but I can add it in case I missed something and it happens

Reply:
so for the last case I updated the api and the code so that if this case ever happen, when the user disables his renewal it will disable it on all the enabled entries for his domain (with a multicall)